composer.json

```diff
@@ -9,10 +9,10 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^5.3|^7.0"
+        "php": "^5.3 | ^7.0"
     },
     "require-dev": {
-        "phpunit/phpunit": ">=3.7.38 <6.0",
+        "phpunit/phpunit": "^4.8 | ^6.5",
```

Comment:
```
* Backported the forward compatibility layer for PHPUnit 6 from PHPUnit 5 so that `PHPUnit\Framework\TestCase` can be used instead of `PHPUnit_Framework_TestCase`
```

if you want to use namespaces, they were implemented only since 4.8.35

Reply:
In ```Composer.json```, define the ```^4.8``` will use and find the latest version.
It will install the version ```4.8.36```.

Reply:
on, any way this directive is root-only and only developer itself may install this dependency